Examples/MappedInDemoV2/BasicWithStoryBoard/MapViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  MapViewController.swift
+//  mapBox-Test
+//
+//  Created by Youel Kaisar on 2020-08-27
+//  Copyright © 2020 Mappedin. All rights reserved.
+//
+
+import UIKit
+import Mapbox
+import Mappedin
+
+class MapViewController: UIViewController{
+    
+    var venueSlug: String? = "the-dubai-mall"
```

Comment:
I think we should use "mappedin-demo-mall" for our examples

Reply:
agreed